bubble-wrap-style.yaml

```diff
@@ -3906,7 +3906,7 @@ layers:
                         priority: 2
                         visible: global.text_visible_admin
                         text_source: global.ux_language_text_source_boundary_lines_left_right
-                        offset: [0, 5px]
+                        offset: [0px, 5px]
```

Comment:
I think omitting units here is ok? There's an example of it in the docs where it describes `offset` with stops.

Reply:
Hmm, the documentation mention **no default** for point `offset` units.

Reply:
And ES is following the docs and fails when no units are specified for an `offset`.